### PR TITLE
Issue #184 documentation

### PR DESCRIPTION
**Pull request by albamig**
This PR contains a first version for the documentation.

### CODE REVIEW

File: AUTHORS.md [hunk 1]
```diff
@@ -10,4 +10,4 @@ Contributors
 ------------
```
Comment (garciparedes) [inline AUTHORS.md:10]: I would add a new section called as Core Devs to include our names, and also keep a `Contributors` one for external contributor (like this case https://github.com/Clariteia/minos-pypackage/blob/main/%7B%7Bcookiecutter.project_slug%7D%7D/AUTHORS.md)

Also, you you forgot to include your own name!!

File: Makefile [hunk 2]
```diff
@@ -86,4 +86,7 @@ dist: clean ## builds source and wheel package
 install:
 	poetry install
 
+update:
```
Comment (garciparedes) [inline Makefile:89]: Good contribution! This will ease the updating process so much!

File: docs/conf.py [hunk 3]
```diff
@@ -134,37 +146,35 @@
 # [howto, manual, or own class]).
 latex_documents = [
     (master_doc, 'minos.tex',
-     'Minos Microservice Networks Documentation',
+     'Minos Microservice Netowrk Documentation',
```
Comment (garciparedes) [inline docs/conf.py:149]: Fast typing my friend? 😏
```suggestion
     'Minos Microservice Network Documentation',
```

File: docs/make.bat [hunk 4]
```diff
@@ -9,7 +9,7 @@ if "%SPHINXBUILD%" == "" (
 )
 set SOURCEDIR=.
 set BUILDDIR=_build
-set SPHINXPROJ=minos_microservice_networks
+set SPHINXPROJ=minos_microservice_common
```
Comment (garciparedes) [inline docs/make.bat:12]: I think this would be referred to networks. I am right? 🤔

File: docs/usage.rst [hunk 5]
```diff
@@ -2,6 +2,8 @@
 Usage
 =====
 
-To use Minos Microservice Networks in a project::
+To use Minos Microservice Common in a project::
+
+    import minos_microservice_common
```
Comment (garciparedes) [inline docs/usage.rst:7]: I think this would be referred to networks. I am right? 🤔

File: docs/make.bat (hunk 4 above)
Comment (albamig) [reply docs/make.bat:12]: Nice catch. I think this is what is breaking the test's pipeline :sweat:

**Pull request merged**